.github/workflows/unittests.yml

```diff
@@ -6,7 +6,6 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
```

Comment:
Why was this removed?

Reply:
Because I wanted to make sure this PR passes unittests. With the branch specified like that, unit tests don't run on stacked PRs.

Reply:
It's in the commit message: [Run unittests on stacked PRs too](https://github.com/pyiron/pyiron_base/pull/1455/commits/8005fe2b1c1f090551b6e222d0aa61cabab04a22)